### ISSUE TITLE
How to add sharepoint group drive to onedrive in docker?

### ISSUE THREAD

**Issue by prestevez** [abraunegg/onedrive#803]
Hi,

I've been running onedrive in docker using the "First run" instructions provided in Docker.md. Everything is running great and I really appreciate the fact that containerised versions of ondrive "just work".

Where I've stumbled is in how to add sharepoint group drives to dockerised onedrive instances. I think I get the gist of how to add the drive ID to the config file as outlined in the Office365.md, but I haven't figured out how to obtain the drive id as the container running onedrive does not offer a command line interface (or at least I haven't been able to figure out how to access it).

I would greatly appreciate some guidance on the matter.

**Comment by abraunegg** [comment]
@prestevez 
The best suggestion here is to run the client outside of a Docker environment to obtain the information required, then configure the Docker container as required.

**Comment by prestevez** [comment]
Thanks for your response. I wanted to avoid installing onedrive locally, to keep things tidy. I managed to run an interactive bash session in a container using the driveone/onedrive image. I used the following:

`docker run -it --name onedrive_bash -v onedrive_conf:/onedrive/conf --entrypoint /bin/bash driveone/onedrive
`

I used the `-v onedrive_conf:/onedrive/conf ` to see if the instance in the bash session would read my configuration and would not need to re-authorise the onedrive client running in that instance. However, that did not work. So the onedrive client in the session had to be authorised using the standard method. (Thus the -v flag option may not be required).

Then I followed the instructions in the Office365.md file and got the drive ids for my sharepoint libraries. Was pretty straightforward.

Now I'll try to sync the libraries to my machine, but I'll probably not do it using my standard onedrive dockerised deamon. I read somewhere in the issues that the sharepoint libraries are synced to the local onedrive dir. Since I want to sync to a different dir, I'll probably run other containers for each sharepoint library. 

Is it possible to *only* sync the drives specified in the drive id config? Thanks for any comments.
